website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,118 @@
+slug: podman-desktop-release-0.10
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+This release note covers Podman Desktop 0.10 release changes.
+
+- **Containers Configuration**: Container creation wizzard allowing to define environment variables, networking and more configuration options.
+- **Kubernetes Improvements**: Play Kubernetes YAML, custom Kubeconfig path support, reload of kube context.
+- **Registries Configuration**: Revamped registries configuration UI.
+- **Podman Version**: Podman 4.3.1 now included in Windows and Mac installers.
+- **UX/UI Improvements**: Improved lists, better contrast on se design,  and more.
+
+Podman Desktop 0.10 is now available. [Click here to download it](/downloads)! 
+
+<!--truncate-->
+_________________
+
+## Release Details
+
+### New configuration wizzard to create containers ([#773](https://github.com/containers/podman-desktop/pull/773))
+
+Until now, we could only specify port binding when building images to start containers. The create container wizzard now includes several options enabling configuring volumes, environment variables, restart policy and settings on networking and security. It is also possible to reuse an existing (and already configured) a network when creating containers.
+
+![new container creation wizard](img/podman-desktop-release-0.10/new_container_creation_wizard.gif)
+
+### Kubernetes Capabilities Improvements
+
+**Custom Kubeconfig file path ([#780](https://github.com/containers/podman-desktop/pull/780))** 
+
+Kubeconfig path location is now configurable from `Settings > Preferences > Kubernetes: Kubeconfig` and can be set to a custom path. By default, Podman Desktop use the path `$HOME/.kube/config` for the Kubeconfig file.
+
+![custom kubeconfig file path](img/podman-desktop-release-0.10/custom-kubeconfig.png)
+
+**Play Kubernetes YAML from the Pods list ([#739](https://github.com/containers/podman-desktop/pull/739))**
+
+Podman Desktop enables to play existing Kubernetes YAML files. This is now available from `Play Kubernetes YAML` button added to the Pods list, in addition to the Containers list. 
+
+![play k8s yaml](img/podman-desktop-release-0.10/play_k8s_yaml.gif)
+
+**Kube context automatically reloaded when updated ([#813](https://github.com/containers/podman-desktop/pull/813))**
+
+Any change to the kube context will now be detected by Podman Desktop in its running state. The system tray will  One can now reload kube context without restarting Podman Desktop.
+
+### Revamped Container Registries UI ([#446](https://github.com/containers/podman-desktop/issues/446))
+
+The registries configuration UI has been revamped. Instead of using tiles for displaying the registries, now it uses a list where each registries can be added or edited directly in the list. The kebab menu provides options to edit or remove a registry from the list. 
```

Comment:
```suggestion
The registries configuration UI has been revamped. Instead of using tiles for displaying the registries, now it uses a list where each registry can be added or edited directly in the list. The kebab menu provides options to edit or remove a registry from the list. 
```